src/features/profile/ProfileSlice.ts

```diff
@@ -0,0 +1,256 @@
+    { value: 'ru', label: 'Russian' },
+    { value: 'ru-mo', label: 'Russian (Moldavia)' },
+    { value: 'sz', label: 'Sami (Lappish)' },
+    { value: 'sg', label: 'Sango' },
+    { value: 'sa', label: 'Sanskrit' },
+    { value: 'sc', label: 'Sardinian' },
+    { value: 'sd', label: 'Sindhi' },
+    { value: 'si', label: 'Singhalese' },
+    { value: 'sr', label: 'Serbian' },
+    { value: 'sk', label: 'Slovak' },
+    { value: 'sl', label: 'Slovenian' },
+    { value: 'so', label: 'Somani' },
+    { value: 'sb', label: 'Sorbian' },
+    { value: 'es', label: 'Spanish' },
+    { value: 'es-ar', label: 'Spanish (Argentina)' },
+    { value: 'es-bo', label: 'Spanish (Bolivia)' },
+    { value: 'es-cl', label: 'Spanish (Chile)' },
+    { value: 'es-co', label: 'Spanish (Colombia)' },
+    { value: 'es-cr', label: 'Spanish (Costa Rica)' },
+    { value: 'es-do', label: 'Spanish (Dominican Republic)' },
+    { value: 'es-ec', label: 'Spanish (Ecuador)' },
+    { value: 'es-sv', label: 'Spanish (El Salvador)' },
+    { value: 'es-gt', label: 'Spanish (Guatemala)' },
+    { value: 'es-hn', label: 'Spanish (Honduras)' },
+    { value: 'es-mx', label: 'Spanish (Mexico)' },
+    { value: 'es-ni', label: 'Spanish (Nicaragua)' },
+    { value: 'es-pa', label: 'Spanish (Panama)' },
+    { value: 'es-py', label: 'Spanish (Paraguay)' },
+    { value: 'es-pe', label: 'Spanish (Peru)' },
+    { value: 'es-pr', label: 'Spanish (Puerto Rico)' },
+    { value: 'es-es', label: 'Spanish (Spain)' },
+    { value: 'es-uy', label: 'Spanish (Uruguay)' },
+    { value: 'es-ve', label: 'Spanish (Venezuela)' },
+    { value: 'sx', label: 'Sutu' },
+    { value: 'sw', label: 'Swahili' },
+    { value: 'sv', label: 'Swedish' },
+    { value: 'sv-fi', label: 'Swedish (Finland)' },
+    { value: 'sv-sv', label: 'Swedish (Sweden)' },
+    { value: 'ta', label: 'Tamil' },
+    { value: 'tt', label: 'Tatar' },
+    { value: 'te', label: 'Teluga' },
+    { value: 'th', label: 'Thai' },
+    { value: 'tig', label: 'Tigre' },
+    { value: 'ts', label: 'Tsonga' },
+    { value: 'tn', label: 'Tswana' },
+    { value: 'tr', label: 'Turkish' },
+    { value: 'tk', label: 'Turkmen' },
+    { value: 'uk', label: 'Ukrainian' },
+    { value: 'hsb', label: 'Upper Sorbian' },
+    { value: 'ur', label: 'Urdu' },
+    { value: 've', label: 'Venda' },
+    { value: 'vi', label: 'Vietnamese' },
+    { value: 'vo', label: 'Volapuk' },
+    { value: 'wa', label: 'Walloon' },
+    { value: 'cy', label: 'Welsh' },
+    { value: 'xh', label: 'Xhosa' },
+    { value: 'ji', label: 'Yiddish' },
+    { value: 'zu', label: 'Zulu' },
+  ],
+  loading: false,
+}
+
+export const pastelProfileSlice = createSlice({
+  name: 'pastelProfile',
+  initialState,
+  reducers: {
+    getLocations(state: TProfileState, action: PayloadAction<TSelect[]>) {
+      state.locations = action.payload
+      state.loading = true
+    },
+    getLocationsFailure(state: TProfileState) {
```

Comment:
Name of function should reflect what function does

Reply:
fixed